src/ngrok.js

```diff
@@ -0,0 +1,77 @@
+// setup ngrok
+import ngrok from "@ngrok/ngrok";
+import dotenv from "dotenv";
+import open from "open";
+import axios from "axios";
+import https from "https";
+
+dotenv.config();
+
+const auth = process.env.NGROK_AUTHTOKEN;
+
+(async function() {
+  try {
+
+    // Check if the Engine has been started
+    const healthCheckUrl = "https://localhost:3005/system/health";
+    const httpsAgent = new https.Agent({  
+      rejectUnauthorized: false // This will skip the SSL verification
+    });
+
+    // Check if the Engine is up and running if not retry after 5 seconds
+    const checkEngineHealth = async () => {
+      try {
+        const healthResponse = await axios.get(healthCheckUrl, { httpsAgent });
+        if (healthResponse.status === 200) {
+          console.log("Engine is up and running.");
+
+          // Check if the auth token is present
+          if (!auth || auth==="" || auth.includes("<") || auth.includes(">")) {
+            console.log("\n===============================================");
+            console.log("No NGROK_AUTHTOKEN found. Your Engine is not accessible from the internet.");
+            console.log("You can view your Engine locally at https://localhost:3005");
+            console.log("To make your Engine accessible from the internet.");
+            console.log("Please provide a valid NGROK_AUTHTOKEN in the .env file and redeploy.");
+            console.log("===============================================\n");
```

Comment:
Whats the use of NGROK_AUTHTOKEN? also can we change it to `NGROK_AUTH_TOKEN` (if needed)